code/modules/vehicles/motorized_wheelchair.dm

```diff
@@ -0,0 +1,153 @@
+/obj/vehicle/ridden/wheelchair/motorized
+	name = "motorized wheelchair"
+	desc = "A chair with big wheels. It seems to have a motor in it."
+	max_integrity = 150
+	var/speed = 2
+	var/power_efficiency = 1
+	var/power_usage = 40
+	var/panel_open = FALSE
+	var/list/required_parts = list(/obj/item/stock_parts/manipulator, 
+							/obj/item/stock_parts/manipulator,
+							/obj/item/stock_parts/capacitor)
+	var/obj/item/stock_parts/cell/power_cell
+
+/obj/vehicle/ridden/wheelchair/motorized/CheckParts(list/parts_list)
+	..()
+	refresh_parts()
+
+/obj/vehicle/ridden/wheelchair/motorized/proc/refresh_parts()
+	var/_temp = 0
+	for(var/obj/item/stock_parts/manipulator/M in contents)
+		_temp += M.rating
+	speed = _temp
+	_temp = 0
+	for(var/obj/item/stock_parts/capacitor/C in contents)
+		_temp += C.rating
+	power_efficiency = _temp
+	var/datum/component/riding/D = GetComponent(/datum/component/riding)
+	D.vehicle_move_delay = round(CONFIG_GET(number/movedelay/run_delay) * 6.7) / speed
```

Comment:
this should max out at the same as run_delay.

Reply:
Right, so for tier 4 and 10 power use every time you move you should not be able to move faster than running speed? It goes slightly over running speed with T3 and T4 manipulators in it, I am more willing to change the power use than the maximum speed.

Reply:
what is 10 power relative to

Reply:
well yeah, its a too low. 1500 turfs can be moved with tier 4 capacitor and 375 with a tier 1 one

Reply:
I don't want to drain the cell too fast because then it is just annoying when you have to replace the cell often. I am thinking around 80 power usage.

Reply:
It should be annoying if you're going mach 5 tbh

Reply:
This was not resolved denton

Reply:
I am actually thinking about scaling power usage with speed because it is not fun when you have to change/charge the cell every time you move 2 turfs with a potato battery taken from beepsky.

Reply:
with 80 power usage and 15000 power cell you can move 750 turfs with tier 4 capacitor and 187 with a tier 1 capacitor.

Reply:
well if no one has anything to say about this having power usage of 100 now then Ill resolve this.